docs/style-guide.md

```diff
@@ -1,30 +1,30 @@
 # Code style guide
 
-The document introduces the style guide for codes submited to poi. Codes of good and coherent style is friendly to other contributors and yourself in the future. Please follow them.
+This document introduces the style guide for code submitted to poi. Good and coherent code style is friendly to other contributors and yourself in the future. Please follow them.
 
-## Make eslint happy
-poi uses [eslint](http://eslint.org/) as code linter, and is bundled with `.eslintrc.js` conf file. Eslint is accessible through [command line](http://eslint.org/docs/user-guide/command-line-interface), or you may try editors with eslint extensions, e.g.
+## Make ESlint happy
+poi uses [ESlint](http://eslint.org/) as code linter, and is bundled with `.eslintrc.js` conf file. ESlint is accessible through [command line](http://eslint.org/docs/user-guide/command-line-interface), or you may try editors with ESlint extensions, e.g.
 + Sublime Text 3 + SublimeLinter + SublimeLinter-eslint
 + Atom + Linter
 + Visual Studio Code + ESlint
 
-You should avoid eslint errors, and leave as few warnings as you can; our rules, however, cannot be 100% perfect, so if you encounter errors that cannot be fixed, please provide the information in code comments or Pull Request.
+You should avoid ESlint errors, and leave as few warnings as you can. Our rules are not perfect, so if you encounter any ESLint errors that cannot be resolved, please provide details of the error in comments or Pull Request.
 
 ## Other styles
 ### Naming
-Game api's variable naming are snake_case (e.g. `api_mst_slotitems`), you're allow to use them, for your variable definition, however, [camelCase](https://en.wikipedia.org/wiki/Camel_case) naming is recommended, for example, `apiMstSlotitems`.
+Game api's style use snake_case (e.g. `api_mst_slotitems`). You are allowed to use them for your variable definition too. However, [camelCase](https://en.wikipedia.org/wiki/Camel_case) is recommended, for example, `apiMstSlotitems`.
 
 ``` javascript
 const {api_maparea_id, api_mapinfo_no} = postBody
 const mapId = `${api_maparea_id}${api_mapinfo_no}`
 ```
-Function naming rules are the same as variables.
+Function use the same naming rules as variables.
 
-for constants, use SNAKE_CASE: `MAX_RETRY`，`APPDATA_PATH`.
+For constants, use UPPERCASE_SNAKE_CASE: `MAX_RETRY`，`APPDATA_PATH`.
 
-For class or react element, use CamelCase: `FileWriter`，`ShipView`.
+For class or react element, use PrimalCase: `FileWriter`，`ShipView`.
```

Comment:
You are right, I knew I could have a better name for it, and it should be Pascal case.